src-script/zap-versionstamp.js

```diff
@@ -18,14 +18,9 @@
 
 const scriptUtil = require('./script-util.js')
 
-let startTime = process.hrtime.bigint()
-
 //workaround: executeCmd()/spawn() fails silently without complaining about missing path to electron
 process.env.PATH = process.env.PATH + ':/usr/local/bin/'
 
-scriptUtil
-  .stampVersion()
-  .then(() => scriptUtil.doneStamp(startTime))
-  .catch((err) => {
-    console.log(err)
-  })
+scriptUtil.stampVersion().catch((err) => {
```

Comment:
Do we not need this time stamp we had before?

Reply:
We need the "stampVersion", but we don't need the "doneStamp".
DoneStamp simply printed out the "All done...." text earlier.
I've now change doneStamp() to also revert the changes to package.json.

In this case, that was unnecessary, so I removed the whole thing, including the "All done..." printout, which seemed to me, honestly, a bit stupid to print out just in case when you are creating the .version file.